Firmware/odrive-interface.yaml

```diff
@@ -211,6 +211,8 @@ interfaces:
           WatchdogTimerExpired:
           MinEndstopPressed:
           MaxEndstopPressed:
+          OverTemp:
+            doc: Check `fet_thermistor.error` and `motor_thermistor.error` for more information.
```

Comment:
let's move this after `HomingWithoutEndstop` to reduce the likelihood of breaking someone's code silently

Reply:
Fixed.